packages/aws-cdk/test/api/lambda-hotswap-deployments.test.ts

```diff
@@ -0,0 +1,351 @@
+import { Lambda } from 'aws-sdk';
```

Comment:
Another comment: can we move all of the hotswap-related test files to a new `test/api/hotswap/` directory?